MANUAL.md

```diff
@@ -19,6 +19,7 @@ Other options:
 ```
   -h, --help            show help message and exit
   -x, --extract         autoextract compressed files
+  -v, --verbose         details on found issues as script runs
```

Comment:
Actually, we removed verbose in favour of --log debug for the 1.0 release!  So for this PR, please drop all the changes made to MANUAL.md.